src/moore/settings/dev.py

```diff
@@ -28,6 +28,16 @@
             'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
         }
     }
+    # Override search backend to not use postgres
+    WAGTAILSEARCH_BACKENDS = {
+        'default': {
+            'BACKEND': 'wagtail.search.backends.db',
+        }
+    }
+    # Kinda hacky, but since this app has postgres-specific migrations
```

Comment:
Skip the "kinda hacky". I find the solution to be a good and simple and it solves the problem. Keep the rest of the comment though so that you understand why the code is there